source/includes/_invoices.md

```diff
@@ -2,22 +2,39 @@
 
 Invoices allow you to store invoice information for later payment.
 
+The Invoice integrates a system of **Invoice Line**. An invoice line is defined by an `amount`, a `currency` and a `description`.
+Our invoice line allows you to have multiple services and/or products on one Invoice.
+
 ## Invoice object
 
 Attributes | Type | Description
 ---------- | ---- | ------
-**user** | `object` | [`User`](#users) associated with invoice
-**amount** | `number` | The invoice amount. Two decimals.
-**currency** | `string` | ISO 4217 code of currency, for example: "EUR"
-**account_number** | `string` | Account number of the issuer that will receive the payment.
+user | `object` | [`User`](#users) associated with invoice
+amount | `number` | The invoice amount. Two decimals.
+currency | `string` | ISO 4217 code of currency, for example: "EUR"
+account_number | `string` | Account number of the issuer that will receive the payment.
 due_date | `object` | The due date of the invoice, `timestamp` format
 issued_date | `object` | The date the invoice was generated, `timestamp` format
 message | `string` | Message (KID) used for paying the invoice as it will appear in the issuer's bank
 issuer | `object`| [`Issuer`](#issuers) of the invoice
 issuer_alias | `string` | The user may want to use an alias for the [`issuer`](#issuers)
 image_url | `string` | The url for the image of the invoice that is returned from the ocr
 status | `string` | The status of the Invoice. Default is CREATED. Additionally there are: "SCHEDULED", "DONE", "FAILED" and "CANCELLED"
+payments | `array` | List of [`Payment`](#payments) objects associated with invoice
+invoice_lines | `array` | List of `Invoice Line` associate with an invoice
+accounting_reference | `string` | Is the ID reference from an Accounting system. If your company is using an Account system supported by ShareActor this field will be automatically setup.
+external_reference | `string` | You can set this field to put your own invoice reference or if you are using a third party.
 
+### Invoice Line
+
+Attributes | Type | Description
+---------- | ---- | ------
+**amount** | `number` | Amount as a number with two decimals. eg., 12.34.
+**currency** | `string` | ISO 4217 code of currency, for example: "EUR"
+**description** | `string` | The description of what the invoice line is. e.g, a shipping line.
+external_reference | `string` | If you are using a 3rd party the `external_reference` could be the ID returns by it.
```

Comment:
If you are using third-party services, the `external_reference` could be their ID